src/components/styledComponents.js

```diff
@@ -145,16 +146,32 @@ export const defaultTheme = createMuiTheme({
         margin: `${spacing(4)} ${spacing(3)} ${spacing(4)} 0px`,
       },
     },
+    MuiIconButton: {
+      root: {
+        padding: '8px 0px 8px 12px',
+      },
+    },
+    MuiMenuItem: {
+      root: {
+        width: '100%',
+        '&:hover': {
+          backgroundColor: `${colors.orange}2f`,
```

Comment:
What is the `2f` doing here?

Reply:
It is the transparency.
I wasn't sure if it is useful to add a new color for that single used

Reply:
Ah, I didn't realize you can add on another pair for rgba values in that format... that's cool!

Reply:
Other comment ?

Reply:
Looks good to merge to me!